src/main/java/sirius/biz/web/BizController.java

```diff
@@ -198,7 +200,11 @@ protected void load(WebContext ctx, BaseEntity<?> entity, List<Mapping> properti
             String propertyName = property.getName();
 
             try {
-                property.parseValue(entity, ctx.get(propertyName));
+                if (property instanceof StringListProperty) {
```

Comment:
maybe a method "parseValueFromWebContext" would be better, what do you guys think?

Reply:
decision on: `property.parseValues(entity, Values.of(ctx.getParameters(propertyName)));`
`parseValues` needs to be implemented and calls `parseValue` with the first value by default